### PR TITLE
IE11 fixes

### PR DESCRIPTION
**Pull request by DanReyLop**
I'm assigning reviewers based on the suggestions, but whoever can take a look in IE11 should be enough.

Browsing wp.com in IE11, I noticed 3 broken things. They are separated by commit.

* https://github.com/Automattic/wp-calypso/commit/dee37fc8eb3ba445daba4f013361b0f37ccbab14: ~A "Syntax Error" message would appear in the console in IE11, because there was an arrow function (`a => b`) that wasn't being transpiled. I know that is throwaway code, but since its purpose is to have some performance metrics, discarding all the user with a super-old IE browser can schew the results.~ Already fixed in `master`. Ignore.
* https://github.com/Automattic/wp-calypso/commit/f79d56f9fa269567e3bb9007943421566729f11e: The "offline mode" (dev env only) logic was using `URLSearchParams`, which isn't supported by IE/Edge. I swapped it for the `qs` module which we are using in other parts of the app. This only affects the dev environment. Before this you would see an obvious error in the console complaining about `URLSearchParams`.
* https://github.com/Automattic/wp-calypso/commit/b9e120657cf3d1afb85f135f4b78aa881d177bd2: When swapping `require('blah')` with `import 'blah'` all through Calypso, this was overlooked. All the `import`s are resolved and run before any other code on the module, even if the code comes before the `import` statement. So the polyfills required for `draft-js` to run in IE11 weren't being loaded on time. Before, the `Settings -> Traffic` page wouldn't even load in IE (complaining about non-existing `String.prototype.somethingIDontRemember`), now at least it loads and everything works.

### CODE REVIEW

File: server/pages/index.pug [hunk 1]
```diff
@@ -262,7 +262,9 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 					}
 
 					var pixel = document.createElement( 'img' );
-					pixel.src = statsPixel + '?' + Object.keys( stats ).map( key => encodeURIComponent( key ) + '=' + encodeURIComponent( stats[ key ] ) ).join( '& ' );
+					pixel.src = statsPixel + '?' + Object.keys( stats ).map( function( key ) {
```
Comment (yurynix) [inline server/pages/index.pug]: Looks good 👍 , sorry about that

**Comment by samouri** [comment]
- https://github.com/Automattic/wp-calypso/commit/b9e120657cf3d1afb85f135f4b78aa881d177bd2 LGTM
- I'm going to investigate https://github.com/Automattic/wp-calypso/commit/f79d56f9fa269567e3bb9007943421566729f11e a bit more.  using nodejs libs in client code usually is a cause for alarm for me as many of them bring in heavy polyfills

**Comment by DanReyLop** [comment]
> I'm going to investigate f79d56f a bit more. using nodejs libs in client code usually is a cause for alarm for me as many of them bring in heavy polyfills

We're already using `qs` in many `client/` files, plus this is only using in the `dev` environment.

**Comment by samouri** [comment]
> We're already using qs in many client/ files, plus this is only using in the dev environment.

Even more reason to investigate then!
All of the polyfills are listed [here](https://github.com/webpack/node-libs-browser).
`qs` pull in https://github.com/mike-spainhower/querystring, (`querystring-es3`) which is only 700B once gzipped. It currently lives in the `build` chunk.  Maybe we'll remove it one day...but not today :)

🚢

**Pull request merged**